tensorzero-core/tests/e2e/howdy.rs

```diff
@@ -0,0 +1,84 @@
+#![expect(clippy::print_stdout)]
+use crate::common::get_gateway_endpoint;
+use reqwest::Client;
+use reqwest::StatusCode;
+use serde_json::json;
+use tensorzero_core::clickhouse::test_helpers::get_clickhouse;
+use tensorzero_core::howdy::{get_deployment_id, get_howdy_report};
+use tokio::time::Duration;
+use uuid::Uuid;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_get_deployment_id() {
+    let clickhouse = get_clickhouse().await;
+    let deployment_id = get_deployment_id(&clickhouse).await.unwrap();
+    println!("deployment_id: {deployment_id}");
+    assert!(!deployment_id.is_empty());
+}
+
+#[tokio::test]
+async fn test_get_howdy_report() {
+    let client = Client::new();
+    let clickhouse = get_clickhouse().await;
```

Comment:
Can you use `get_clean_clickhouse`, and construct an embedded client from the `ClickHouseConnectionInfo`? This will let us assert the exact number of inferences